cli/util.go

```diff
@@ -155,10 +155,10 @@ func writeOutput(output io.Writer, input interface{}) error {
 	return nil
 }
 
-// newRemoteClient returns a remote client that connects to the service at the
+// newRemoteManager returns a remote manager that connects to the service at the
```

Comment:
For consistency it seems like you use `remote.Manager` elsewhere

Reply:
Done.